newIDE/app/src/EventsSheet/SelectionHandler.js

```diff
@@ -105,6 +105,18 @@ export const hasInstructionSelected = (selection: SelectionState): boolean => {
   return !!Object.keys(selection.selectedInstructions).length;
 };
 
+export const hasSelectedAtLeastOneCondition = (
+  selection: SelectionState
+): boolean => {
+  for (let instructionContext of getSelectedInstructionsContexts(selection)) {
+    if (instructionContext.isCondition) {
+      return true;
+      break;
```

Comment:
You don't need a break after a return, return will end the current function ;)

Reply:
oooh *slaps forehead* xD

Reply:
sorry, late night coding silliness again

Reply:
Flow was good enough this time to let you know about this ;)
![image](https://user-images.githubusercontent.com/1280130/46382085-235afd00-c6a1-11e8-9a6d-3d872fd12dc0.png)